backend/src/server.ts

```diff
@@ -0,0 +1,16 @@
+// API Server logic goes here 
+// It's the first file called when running this API
+// It hands HTTP Requests to ./express.ts
+import app from './express';
+import dotenv from 'dotenv';
+
+// Get environment variables from .env file using DotEnv
+dotenv.config();
+
+// Get api port from environment variables, failsafe to 3000
+const API_PORT = process.env.API_PORT || 3000;
+
+// Start HTTP Server
+app.listen(API_PORT, () => {
+  console.info(`Server running at http://localhost:${API_PORT}`);
```

Comment:
```suggestion
  console.info(`[Server] running on port ${API_PORT}`);
```
You don't have to phrase it like this, its just the last thing to clean up :)